Backend/dev/migrations/dynamic-offer-driver-app/0140-adding-merchantId-field-to-special-location-table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_driver_offer_bpp.special_location ADD COLUMN merchant_id character(36) NOT NULL REFERENCES atlas_driver_offer_bpp.merchant (id) default 'favorit0-0000-0000-0000-00000favorit';
```

Comment:
instead of giving a default merchant_id here, can you do add column without constraint first then insert specific merchant and add not null constraint after that ?

Reply:
okay